### PR TITLE
#25 Upgrade to .NET 7.0

### PR DESCRIPTION
**Pull request by fedotkin**

### CODE REVIEW

File: trey-nash/Program.cs [hunk 1]
```diff
@@ -1,27 +1,24 @@
 ﻿using Fedotkin.Dotnet.TreyNash.Ch5_InterfacesAndContracts;
-class Program
+//TODO
+//namespace Fedotkin.Dotnet.TreyNash.General;
```
Comment (raman-m) [inline trey-nash/Program.cs:3]: ## 1<sup>st</sup> issue
### ${\color{red}TODO\ has\ no\ description!}$

You have to create the task/issue and the title should be here as a text after TODO!

## 2<sup>nd</sup> issue
### ${\color{orange}Namespace\ has\ extra\ 'General'\ suffix!}$
The main program class should not be in any sub namespaces! So, it should be defined in root namespace!
The root namespace should end with the Solution name which is "**TreyNash**" in this solution!

File: trey-nash/Program.cs [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
+// Select book chapter and run the demo
+switch (chapterNo)
+{
+    case 5:
+        Chapter5.Run();
+        break;
+    default:
+        Console.WriteLine("Chapter {0}: Sorry, there are no exercises and no implemented solutions to demonstrate!", chapterNo);
+        break;
 }
```
Comment (raman-m) [inline trey-nash/Program.cs:24]: I'm not sure, but, do we have new line after closing curly bracket?
Each source code file must end with new line char aka '**\n**'!

File: trey-nash/ChapterExercises.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
```
Comment (raman-m) [inline trey-nash/ChapterExercises.csproj:6]: ### ${\color{orange}You\ don't\ use\ global\ usings!}$
What implicit **usings** did you define in the solution at the moment?
Create **global usings** file and define at least the `System` namespace in this file, like that:
```csharp
global using System;
```

**Review by raman-m — CHANGES_REQUESTED** [review]
Fix the issues, plz!

File: trey-nash/Program.cs (hunk 1 above)
Comment (raman-m) [reply trey-nash/Program.cs]: Please note that TODO line should have new line before TODO comments block!

File: trey-nash/Program.cs (hunk 2 above)
Comment (fedotkin) [reply trey-nash/Program.cs:24]: Empty line is here

Comment (raman-m) [reply trey-nash/Program.cs:24]: :ok:  Not an issue!

**Review by raman-m — APPROVED** [review]
```diff
+ Approved 
```
⚠️  Use "**Squash and merge**" option.

**Pull request merged**